arwen/context/arwen.go

```diff
@@ -733,6 +770,8 @@ func (host *vmContext) Transfer(destination []byte, sender []byte, gasLimit uint
 	senderAcc.BalanceDelta = big.NewInt(0).Sub(senderAcc.BalanceDelta, value)
 	destAcc.BalanceDelta = big.NewInt(0).Add(destAcc.BalanceDelta, value)
 	destAcc.Data = append(destAcc.Data, input...)
+
+	// TODO set gasLimit or accumulate gasLimit?
```

Comment:
gasLimit can be deleted here.

Reply:
`asyncCall` requires `gasLimit`, in case the call cannot be executed synchronously.